response_operations_ui/views/messages.py

```diff
@@ -195,6 +199,38 @@ def view_selected_survey(selected_survey):
                                response_error=True)
 
 
+@messages_bp.route('/close-conversation', methods=['GET', 'POST'])
+@login_required
+def close_conversation():
+    try:
+        messages = session['messages']
+        selected_survey = session['messages_survey_selection']
+        thread_id = messages[0]['thread_id']
+    except KeyError:
```

Comment:
Should these be in different try/except blocks? I'm guessing that it doesn't matter which of the three dict access calls throw KeyError which is why it is written as one block?

Reply:
Happy with the explanation for this 👍